packages/orbit-components/src/Inline/consts.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+export const ALIGN = ["start", "end", "center"];
```

Comment:
Can we convert all arrays do objects? We use it everywhere else and it's better in the usage in my opinion.
Instead `ALIGN[0]` to use `ALIGN.START`, since I don't remember what's the first in the array.